doc/files.md

```diff
@@ -81,6 +81,7 @@ Subdirectory | File(s)           | Description
 ## GUI settings
 
 `bitcoin-qt` uses [`QSettings`](https://doc.qt.io/qt-5/qsettings.html) class; this implies platform-specific [locations where application settings are stored](https://doc.qt.io/qt-5/qsettings.html#locations-where-application-settings-are-stored).
+You can change this with the `-guisettings=<path>` option.
```

Comment:
Could mention that `<path>` is a directory where the `.ini` file is stored.

Reply:
ACK